pkgs/data/fonts/droid/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "font-droid-${version}";
+  version = "2015-12-09";
+
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/google/fonts/raw/master/apache/droidsans/DroidSans.ttf";
```

Comment:
Is there no other source than github ? Pointing to /raw/master means that the sha256 is going to change whenever upstream is changing the font. Alternatively pick a fixed reference instead of `master`